smartgymapi/handlers/cardio_activity.py

```diff
@@ -0,0 +1,80 @@
+import logging
+from datetime import datetime
+
+from marshmallow import ValidationError
+from pyramid.httpexceptions import HTTPBadRequest, HTTPInternalServerError
+from pyramid.view import view_defaults, view_config
+
+from smartgymapi.models import persist, commit, rollback, delete
+from smartgymapi.lib.factories.cardio_acitivty import CardioActivityFactory, is_cardio_activity_active
+from smartgymapi.lib.validation.cardio_activity import CardioActivitySchema
+from smartgymapi.models.cardio_activity import CardioActivity
+
+log = logging.getLogger(__name__)
+
+
+@view_defaults(containment=CardioActivityFactory,
+               permission='public',
```

Comment:
Permissions should not be public. Check for a different permission and assign this permission to authenticated users.